### ISSUE TITLE
[enhancement][UI][config] Forge Mod Config tie-in

### ISSUE THREAD

**Issue by Cat09** [Lycanite/LycanitesMobs#155]
Dear Lycanite,
   this is just a formal tag for it really, but using the forge mod config can be very useful for adaptively tweaking settings.
I am unsure how difficult this is to build in, but I feel that it's a nice worthwhile feature in the mods that have it.
   Thank you again for your time, Cat09.

**Comment by Lycanite** [comment]
Yeah I've been thinking about adding it for a while, it shouldn't be too difficulty to add, just time consuming, but I'd definitely like to add it.

**Comment by Boolyman** [comment]
If the regular configs don't work, I'm not sure how an in game config tweak would be any better.  As it stands right now the configs just reset back to default after you load the game, no matter what changes you make.

**Comment by Lycanite** [comment]
You need to set "loadDefault" to false in order to stop them from loading default values, this is so that I can make changes to the default values without worrying about people having to manually reset their configs and without worrying about overwriting any changes made by others either.
THe configs will also reset if the json is invalid, an error will show in the log about this. You can also use these commands to reload configs in game:
/lm creatures reload
/lm dungeons reload
/lm spawners reload
/lm mobevent reload

**Comment by xCubit** [comment]
@Lycanite  Where is the "loadDefault" setting located? Do I have to write it in? I have looked around the lycanites config files for a while and have no idea where to find it. 

Thanks!

**Comment by Lycanite** [comment]
You can find it in every individual json file, each file has it's own that why you can 'unlock' a file for editing while the others auto update with each mod update.
